src/mono/wasm/debugger/debugger.md

```diff
@@ -0,0 +1,28 @@
+# Mono Wasm Debugger
+
+## Native debugging
+
+It's possible to debug native code and managed code using the `BrowserDebugProxy` project.
+
+Steps:
+- Install C/C++ DevTools Support (DWARF) extension on chrome. https://chrome.google.com/webstore/detail/cc%20%20-devtools-support-dwa/pdcpmagijalfljmkmjngeonclgbbannb
+- Enable DWARF support: Open DevTools, click on the settings, click on experiments and enable WebAssembly Debugging: Enable DWARF support.<br>
+![image](https://user-images.githubusercontent.com/4503299/170745664-fc7d185c-469c-4443-9c57-545bd79588b8.png)
+- Start the WebAssembly App Without Debugging from VS or dotnet run on command line.
+- Run chrome using this startup parameter: --remote-debugging-port=9222
+- Go to your Blazor App Page
+- Press Ctrl-Alt-D (windows) it will open the debugger page
+- It will show something like this in the Address Bar: ``http://localhost:9222/devtools/inspector.html?ws=127.0.0.1:9300/devtools/page/97FCDA5A332CA3B72031790B26A264EF``
```

Comment:
```suggestion
- It will show something like this in the address Bar: ``http://localhost:9222/devtools/inspector.html?ws=127.0.0.1:9300/devtools/page/97FCDA5A332CA3B72031790B26A264EF``
```